drivers/usb/device/usb_dc_nrfx.c

```diff
@@ -204,25 +201,28 @@ K_MEM_POOL_DEFINE(fifo_elem_pool, FIFO_ELEM_MIN_SZ, FIFO_ELEM_MAX_SZ,
 #define EP_ISOIN_INDEX CFG_EPIN_CNT
 #define EP_ISOOUT_INDEX (CFG_EPIN_CNT + CFG_EP_ISOIN_CNT + CFG_EPOUT_CNT)
 
+#define EP_BUF_DEFAULT_SZ	64UL
+#define ISO_EP_BUF_DEFAULT_SZ	1024UL
```

Comment:
I am not convinced that this makes these macro names less confusing. The sizes are actually the maximum values that can be handled by the hardware (for ISO endpoints it is actually 1023, but I guess it is rounded here, so that the blocks in the memory pool can be partitioned well), so IMO `MAX` in their names is more appropriate than `DEFAULT`.

The actual confusion is generated by the macros `EP_BUF_MIN_SZ` and `EP_BUF_MAX_SZ` which are used only in the definition of the corresponding memory pool but their names may suggest some other purpose. I'd suggest to just rename these macros to `EP_BUF_POOL_BLOCK_MIN_SZ`, `EP_BUF_POOL_BLOCK_MAX_SZ`, and similarly rename other related macros: `EP_BUF_POOL_BLOCK_COUNT` and `EP_BUF_POOL_ALIGNMENT`.

Reply:
okay